lib/webrtc/getstats.js

```diff
@@ -10,9 +11,246 @@ const isFirefox = guess === 'firefox';
 const isSafari = guess === 'safari';
 
 const chromeMajorVersion = isChrome ? guessVersion.major : null;
+const log = new Log('webrtc', 'getStats');
```

Comment:
If we're creating a new log instance, we need to use the loggerName and loglevels passed from connect(). I'm expecting logger initialization would like something like [this](https://github.com/twilio/twilio-video.js/blob/master/lib/connect.js#L213), or [this](https://github.com/twilio/twilio-video.js/blob/master/lib/signaling/v2/peerconnection.js#L147). If it's too much work at this time, I'm ok putting this work post beta.